### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by nancylee713**
## What's this PR do?
- Modify `travis yml`
- Specify repo & branch name
- Add script to run tests and migration
- As of now, staging database has 4 tables
  - `users` (empty)
  - `users_profile` (empty) 
  - `speech_questions` (15 questions loaded -- text, level, and image_url)
  - `grammar_questions` (15 questions loaded -- grammatically incorrect sentences)

### Where should the reviewer start?
- `.travis.yml`

### How should this be manually tested?
- `https://language-learner-be.herokuapp.com/speech_questions`
- `https://language-learner-be.herokuapp.com/grammar_questions`

### What are the relevant tickets?
#33 , #31 

#### Screenshots (if appropriate)
- get all speech questions
Screen Shot 2020-01-06 at 5 33 38 PM

- get all grammar questions
Screen Shot 2020-01-06 at 5 33 47 PM

#### References
- https://docs.travis-ci.com/user/deployment/heroku/
- https://github.com/realpython/discover-flask/blob/master/manage.py

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export APP_SETTINGS="config.DevelopmentConfig"
 export DATABASE_URL="postgresql://localhost/language_learner_dev"
+export UNSPLASH_ACCESS_KEY="62365b47db82a7cf5c2557d8f6b2d2b5627c092b4304a12fdc70dccfd77547e1"
```
Comment (sschipke) [inline .env]: Should this be visible in the repo?

File: .gitignore [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+.env
+
 __pycache__
 
 .vscode
 
 migrations
 
-.env
-
```
Comment (sschipke) [inline .gitignore]: Did we remove it and then add it?

File: .travis.yml [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
 install:
-  - pip install -r requirements.txt
-
+- pip install -r requirements.txt
+before_script:
+- export APP_SETTINGS="config.DevelopmentConfig"
+- export DATABASE_URL="postgresql://localhost/language_learner_dev"
 script:
-  - pytest
+- python3 manage.py db upgrade
```
Comment (sschipke) [inline .travis.yml]: It didn't like this command in the build

File: migrations/script.py.mako [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
 down_revision = ${repr(down_revision)}
+branch_labels = ${repr(branch_labels)}
+depends_on = ${repr(depends_on)}
 
-from alembic import op
-import sqlalchemy as sa
-${imports if imports else ""}
 
 def upgrade():
     ${upgrades if upgrades else "pass"}
```
Comment (sschipke) [inline migrations/script.py.mako]: Do we manually control this file?

File: .env (hunk 1 above)
Comment (nancylee713) [reply .env]: Not sure why this file is being tracked; i'll check

File: migrations/script.py.mako (hunk 4 above)
Comment (nancylee713) [reply migrations/script.py.mako]: Nope; I believe it's auto-generated

File: .travis.yml (hunk 3 above)
Comment (nancylee713) [reply .travis.yml]: I'll remove the line and push it again to heroku-staging

**Pull request merged**